testing/run_tests.py

```diff
@@ -11,6 +11,8 @@
 import glob
 import os
 import re
+import resource
```

Comment:
Looks like this doesn't exist on Windows.

Reply:
Fixed this to only set the core dump ulimit on Linux

(In theory this approach could also work on macOS.  But the directory that holds core dumps on macOS is not writable by the jobs on the LUCI bots).